Repository/RevisionRepository.php

```diff
@@ -558,30 +558,23 @@ public function deleteRevisions(ContentType $contentType = null)
         }
     }
 
-    /**
-     * @param ContentType $contentType
-     * @param \DateTime $until
-     * @param string $by
-     * @param bool $force
-     *
-     * @param bool $id
-     * @return int affected rows
-     * @throws \Exception
-     */
-    public function lockRevisions(ContentType $contentType, \DateTime $until, $by, $force = false, $id = false)
+    public function lockRevisions(?ContentType $contentType, \DateTime $until, $by, $force = false, $id = false): int
```

Comment:
nullable parameters at the end, if this does creates any BC breaks ....

Reply:
Unfortunately the function was already use somewhere else before my refactor (my bad!)

Reply:
No problem!